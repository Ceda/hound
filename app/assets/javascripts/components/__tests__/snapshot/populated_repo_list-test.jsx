import PopulatedRepoList from '../../populated_repo_list.js';

it('renders a list of repos appropriately', () => {
  const organizations = [
    { id: 1, name: "Test org" }
  ]
  const repos = [
    {
      id: 1,
      name: "Test repo",
      owner: {
        id: 1
      }
    }
  ]

  const onRepoClicked = jest.genMockFunction();

  const wrapper = shallow(
    <PopulatedRepoList
      repos={repos}
      onRepoClicked={onRepoClicked}
      isProcessingId={null}
      filterTerm={""}
    />
  );
  expect(wrapper).toMatchSnapshot();
});
