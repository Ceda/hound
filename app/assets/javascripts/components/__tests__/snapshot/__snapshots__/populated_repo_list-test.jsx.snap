exports[`test renders a list of repos appropriately 1`] = `
<ul
  className="repos">
  <li
    className="repo">
    <div
      className="repo-name">
      Test repo
    </div>
    <div
      className="repo-activation-toggle">
      <div
        className="repo-restricted">
        Only repo admins can activate
      </div>
    </div>
  </li>
</ul>
`;
